scripts/lib/install-yarn

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+version=1.22.10
+sha256=05a22fff30d7d8e8005bed277bf20d55111ba2bed65a6b91a0fcd1307b71fd8d
+tarball="yarn-$version.tgz"
+
+check_version() {
+    link="$(command -v yarn)" \
+        && bin="$(readlink -f "$link")" \
+        && current_version="$(jq -r '.version' "${bin%/*/*}/package.json")" \
+        && [ "$current_version" = "$version" ]
+}
```

Comment:
Can you add some comments for this?

Reply:
Not sure what you wanted a comment about, so I guessed.

Reply:
Yeah, that's what I was looking for, thanks!